orderer/common/channelparticipation/validator.go

```diff
@@ -31,10 +29,6 @@ func ValidateJoinBlock(configBlock *cb.Block) (channelID string, err error) {
 		return "", errors.New("invalid block: does not have metadata")
 	}
 
-	if !bytes.Equal(protoutil.BlockDataHash(configBlock.Data), configBlock.Header.DataHash) {
-		return "", errors.New("invalid block: Header.DataHash is different from Hash(block.Data)")
-	}
-
```

Comment:
Why did you remove this test completely and not replace it with the new call? I agree that well formedness does not need to be checked after protoutil.IsConfigBlock but what abouts data hash vs the header.DataHash?

e.g.
```
if !bytes.Equal(protoutil.ComputeBlockDataHash(configBlock.Data), configBlock.Header.DataHash) {
		return "", errors.New("invalid block: Header.DataHash is different from Hash(block.Data)")
	}
```

Reply:
You're right, my bad.